packages/example-getting-started/docs/1-prerequisites-and-setup.md

```diff
@@ -1,34 +1,52 @@
-## Prerequisites
+## Setup
 
 Before we can begin, you'll need to make sure you have some things installed:
-- [Node.js](https://nodejs.org/en/) at v6.x or greater
+- [Node.js](https://nodejs.org/en/) at v8.x or greater
 
 Additionally, this tutorial assumes that you are comfortable with
 certain technologies, languages and concepts.
 - JavaScript (ES6)
 - [npm](https://www.npmjs.com/)
```

Comment:
`npm` is most definitely a prerequisite, IMO

At no point have we tested our application using `yarn`, and unless we do, we shouldn't tell others that it works.

I highly doubt yarn will cause problems, but why would we tell people when we haven't tried it at least once to be sure?

Reply:
With respect to the `REST` portion being a prerequisite, I would say that this tutorial does "gloss over" some of the finer points of a REST API.

Not knowing anything about what REST is, or what HTTP verbs are and how to use them would mean that large portions of the end result of the tutorial would escape some beginners. ("What the heck is POST? Do I just type POST in front of the URL in Chrome/Firefox/Safari/etc?")

Granted, we do tell users to make use of the API Explorer, but knowing why it's not _required_ for testing your API or interfacing with it is important, though it may seem obvious to you and I.

There's a lot of background knowledge required to understand REST so that you can get the most out of this tutorial, as well as see the value of LoopBack from an API point of view.